homeassistant/components/tado/repairs.py

```diff
@@ -17,10 +17,9 @@ def manage_water_heater_fallback_issue(
     integration_overlay_fallback: str | None,
```

Comment:
Martin said in #119526:
> Why do we need this helper to be aware of the entity? We could just pass a list of zone names.

I think he meant:
Why does `manage_water_heater_fallback_issue()` need to be given a list of `water_heater_entities`?